newsletter/2020-03.md

```diff
@@ -0,0 +1,165 @@
+finalize at the end of March, pending feedback.
+
+## API linter
+
+As many readers know, we have published an [API linter][], which is able to
+check much of the guidance provided in API Improvement Proposals for APIs
+defined in protocol buffers.
+
+The linter now has over 150 rules, and is running on many new and existing APIs
+at Google, hopefully pushing us toward greater consistency.
+
+[api linter]: https://github.com/api-linter
+
+## AIPs under review
+
+### AIP-153: Import and export
+
+> Many users want to be able to load data into an API, or get their existing
+> data out of an API. This is particularly important for enterprise users, who
+> are often concerned about vendor lock.
+
+[Import and export][aip-153] are particularly important concepts for enterprise
+APIs, although they are increasing in importance for consumer and SaaS APIs as
+well.
+
+We have actually had a reasonably consistent import and export story in Google
+Cloud for a long time, but it has largely been passed along by tribal knowledge
+("go look at X and copy what they did"). Therefore, this AIP attempts to codify
+what we believe is a long-standing practice.
+
+**Summary:** [AIP-153][] mandates that both import and export methods return
+[long-running operations][aip-151]. Unlike most API methods, they do support
+partial failures, which are handled through the LRO metadata. Finally, they
+have a specified place for per-source and per-destination configuration,
+distinct from configuration for the data itself.
+
+We are aiming to approve AIP-153 at the end of March. If you have feedback,
+please [leave a comment](https://github.com/googleapis/aip/pull/422).
+
+[aip-151]: ../aip/0151.md
+[aip-153]: ../aip/0153.md
+
+### AIP-162: Resource revisions
+
+> Some APIs need to have resources with a revision history, where users can
+> reason about the state of the resource over time.
+
+[Resource revisions][aip-162] is a concept we are seeing crop up more and more,
+and we are seeing an increasing number of situations where audit logging is a
+necessary but insufficient solution. Several APIs have struggled with how to
+model a resource's history over time, and handle common use cases like commit
+and rollback, which is what prompted this AIP.
+
+This is new guidance, put together by the AIP team as well as several API teams
+who needed this functionality near the end of 2019. It has been positively
+received by the teams that needed it, and so we would like to codify it.
+
+**Summary:** [AIP-162][] mandates the use of revision IDs that are appended to
+a resource name with the `@` character for signifying revisions, and sets out
+method structures for most common operations (listing revisions, rollback,
+etc.). It also proposes a tagging mechanism for user-friendly revision names.
+
+We are aiming to approve AIP-162 at the end of March. If you have feedback,
+please [leave a comment]().
+
+[aip-162]: ../aip/0162.md
+
+### AIP-163: Validating changes
```

Comment:
We should probably call this "Change validation" (keep titles in the form of a noun, right?)

Reply:
Done.